### ISSUE TITLE
Suggest to use Cache-Control: immutable for assets with long TTL

### ISSUE THREAD

**Issue by macbre** [macbre/phantomas#683]
* https://hacks.mozilla.org/2017/01/using-immutable-caching-to-speed-up-the-web/ - released with Firefox 49
* https://bitsup.blogspot.com/2016/05/cache-control-immutable.html
* https://www.keycdn.com/blog/cache-control-immutable/

### A header

```
Cache-Control: max-age=365000000, immutable
```

### An example

```
$ curl -sI 'https://www.facebook.com/rsrc.php/v3/yf/r/jPdPvgEM9d8.css' | grep 'Cache'
Cache-Control: public,max-age=31536000,immutable
```

### Tests results

* https://bugs.chromium.org/p/chromium/issues/detail?id=611416#c6 from bbc.co.uk

### Metrics

phantomas will log the following:

```
12:19:42.982 caching: Cache-Control: immutable used for <https://securepubads.g.doubleclick.net/gpt/pubads_impl_118.js>
```

and report `cachingUseImmutable` metric:

```
Offenders for cachingUseImmutable (25):
 * https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js cached for 31536000 s
....
```
